### PR TITLE
Propagate datepicker events up the chain

### PR DESCRIPTION
**Pull request by averted**
This should provide `Event` obj to `onChange` etc.. (Related to #488).

### CODE REVIEW

File: src/calendar.jsx [hunk 1]
```diff
@@ -94,8 +94,8 @@ var Calendar = React.createClass({
     })
   },
 
-  handleDayClick (day) {
-    this.props.onSelect(day)
+  handleDayClick (day, ev) {
```
Comment (martijnrusschen) [inline src/calendar.jsx]: I guess `ev` is the event here? Would you mind changing that to `event`?

Comment (averted) [inline src/calendar.jsx]: Changed `ev` everywhere in PR into `event`.

**Comment by kiddo13** [comment]
when this pull request will be merged?

**Comment by kiddo13** [comment]
I could not get this update from npm install.

**Comment by iknowcss** [comment]
I believe the change won't be available until a new version is published to npm (maybe v0.30.0)

**Comment by martijnrusschen** [comment]
We'll release a new version soon.

**Pull request merged**